ui/components/app/nft-details/index.scss

```diff
@@ -112,7 +112,7 @@ $spacer-break-small: 16px;
   }
 
   &__send-button {
-    margin-inline-end: 8px;
+    margin: 8px;
```

Comment:
I think you can remove this margin completely here

Reply:
Updating it to 4px still looks broken
![Screenshot 2023-07-25 at 12 59 39 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/6c1082aa-e9f7-4d4e-8c75-459fb14dc9b6)